.github/workflows/deploy_ghpages.yml

```diff
@@ -22,7 +22,6 @@ jobs:
             apt-get install -y gcc git libopenmpi-dev openmpi-bin
             python -m pip install --upgrade pip
             python -m pip install -e .[doc,dicodile]
-            python -m pip install git+https://github.com/mne-tools/mne-python.git@main
```

Comment:
But with this, we don't install mne anymore no?

Reply:
mne exists in `install_requires` section in `setup.cfg`.

Reply:
you are right, sorry for the noise.